recipes/inspector/meta.yaml

```diff
@@ -1,16 +1,23 @@
-{% set version = "1.2" %}
+{% set version = "1.3" %}
+
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
 
 package:
   name: inspector
   version: {{ version }}
 
 source:
   url: https://github.com/ChongLab/Inspector/archive/refs/tags/inspector-v{{ version }}.tar.gz
-  sha256: 6d7ca6b22053d06105a549a064cf9a26efaf5c4a2a1a4502f2f36a4eb248ec4a
+  sha256: 6e1278de333990005a720a85048581783040ed4473b2247d0aa0eeaa8cb9ffda
 
 build:
   number: 0
   noarch: generic
+  run_exports:
+    - {{ pin_subpackage('inspector', max_pin="x.x") }}
```

Comment:
Are you sure the max_pin value is "x.x"?

Reply:
We will be releasing versions like 1.2, 1.3, 1.4, 2.1, 3.2, etc. Will max_pin="x.x" suffice? Thanks.

Reply:
See https://github.com/bioconda/bioconda-recipes/blob/master/.github/PULL_REQUEST_TEMPLATE.md#instructions-for-avoiding-api-abi-and-cli-breakage-issues
If you follow Semantic versioning then you should use `"x"`

Reply:
Thanks again. I have changed it to "x".